extensions/devdocs/package.json

```diff
@@ -6,13 +6,15 @@
   "description": "Search through DevDocs documentation sets from Raycast.",
   "icon": "devdocs.png",
   "author": "pomdtr",
+  "contributors": ["djpowers"],
   "commands": [
     {
-      "name": "search-docsets",
-      "title": "Search Docsets",
+      "name": "search-documentations",
```

Comment:
Can we keep the old name here? It's the unique ID of the command where ranking, aliases and hotkeys are saved

Reply:
@djpowers 🙂

Reply:
Hello! Busy day today, have since reviewed. So changing the `name` property would lose the link to existing aliases/hotkeys/rankings? I'd prefer to have it match the new title, but if there's otherwise no migration path to avoid a breaking change then I'm fine keeping the original name.

Will make that change along with some other cleanup.